src/sentry/static/sentry/app/components/eventOrGroupTitle.jsx

```diff
@@ -5,7 +5,14 @@ import {Metadata} from '../proptypes';
 class EventOrGroupTitle extends React.Component {
   static propTypes = {
     data: PropTypes.shape({
-      type: PropTypes.oneOf(['error', 'csp', 'default']).isRequired,
+      type: PropTypes.oneOf([
```

Comment:
kind of painful maintaining these like this, but we can refactor in the future

Reply:
yeah we can move this into our types module later on